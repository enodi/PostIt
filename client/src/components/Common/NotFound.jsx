import React from 'react';
import { Link } from 'react-router';

const NotFound = () => (
  <div className="container center">
    <h1>404 NOT FOUND</h1>
    <p>This Page doesn't exist.
    Click the link below to return back to the home page</p>
    <Link to="/">HOME</Link>
  </div>
);

export default NotFound;
