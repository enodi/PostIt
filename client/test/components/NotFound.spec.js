import React from 'react';
import { shallow } from 'enzyme';

import NotFound from '../../src/components/Common/404NotFound.jsx';

describe('Not Found Component', () => {
  it('should render NotFound Component', () => {
    const wrapper = shallow(<NotFound />);
    expect(wrapper).toMatchSnapshot();
  });
});
