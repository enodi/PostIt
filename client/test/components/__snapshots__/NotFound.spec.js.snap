// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not Found Component should render NotFound Component 1`] = `
<div
  className="container center"
>
  <h1>
    404 NOT FOUND
  </h1>
  <p>
    This Page doesn't exist. Click the link below to return back to the home page
  </p>
  <Link
    onlyActiveOnIndex={false}
    style={Object {}}
    to="/"
  >
    HOME
  </Link>
</div>
`;
